build/cmake/functions.cmake

```diff
@@ -297,13 +301,14 @@ function(wx_set_target_properties target_name is_base)
     target_include_directories(${target_name}
         BEFORE
         PUBLIC
-            ${wxSETUP_HEADER_PATH}
-            ${wxSOURCE_DIR}/include
+            $<BUILD_INTERFACE:${wxSETUP_HEADER_PATH}>
+            $<BUILD_INTERFACE:${wxSOURCE_DIR}/include>
+            $<INSTALL_INTERFACE:include>
```

Comment:
Is `setup.h` installed under `include` by CMake? I'm surprised that we don't need to mention the `lib/wx` subdirectory here.

Reply:
It probably is, I'l add it.